charts/pega/templates/_pega-deployment.tpl

```diff
@@ -289,6 +295,10 @@ spec:
         - name: {{ template "pegaKerberosConfig" }}-config
           mountPath: "/opt/pega/kerberos"
 {{- end }}
+{{- if (.root.Values.hazelcast.encryption.enabled) }}
+        - name: hz-encryption-secrets
+          mountPath: "/opt/hazelcast/certs"
```

Comment:
I assume this path is available in HZ image.
`/opt/hazelcast/certs`

Reply:
yes @Saurabh-16